src/Chapter1.hs

```diff
@@ -478,7 +478,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
```

Comment:
Nice work on specifying a polymorphic type! We don't explain them, so `Int -> Int` will work for now. But that's nice :slightly_smiling_face: